test/compilable/test15371_types.d

```diff
@@ -0,0 +1,7 @@
+module compilable.test15371_types;
+
+//This needs to be in a seperate module to trigger this issue.
+public struct type_test15371 {
+    public int publicField;
+    private int privateField;
+}
```

Comment:
* Files that doesn't contain tests themselves but are required by other tests should be located in `imports`.
* Missing a newline at the end of the file